schema/roles.json

```diff
@@ -32,7 +33,8 @@
             "replace_node": ["list"],
             "health": ["list"],
             "security": ["read"],
-			"cluster_management": ["create", "list"]
+            "cluster_management": ["create", "list"],
+            "s3_iam_users": ["create"]
```

Comment:
Question: we already have s3iamusers access rights: do we need new one?

Reply:
Old- s3iamusers access right is related to s3 user role, here I'm giving permissions to admin and manage user role. 
Moreover In the upcoming sprints we are going to clean up "legacy-s3" code, so old one will get deleted.

Reply:
@avrnv can you please resolve this ? If you want, we can discuss on it.
Thanks